Design: 변경 로그인, splash CSS

parkmihyunn opened the pull request:
1. placeholder 부분 약간 변경했습니다.

1. splash css 수정 하면서 다른 페이지들 가장 바깥에 속성을 부여하였습니다.
2. 3초가 너무 긴 것 같아서 임의로 1.5초로 줄여놓았습니다. 회의 후 정해주세요.

parkmihyunn committed:
Design: 변경 splash css

1. splash css 수정 하면서 다른 페이지들 가장 바깥에 속성을 부여하였습니다.
2. 3초가 너무 긴 것 같아서 임의로 1.5초로 줄여놓았습니다. 회의 후 정해주세요.

parkmihyunn committed:
Design: 변경 로그인 CSS

placeholder 부분 약간 변경했습니다.

hazzun commented on src/App.jsx:
```diff
@@ -49,7 +49,7 @@ function App() {
         <Route path='/' element={loading ? <Splash /> : <Login />}></Route>
       </Routes>
       <Header title={title} back={backHandler} visiable={visiable} />
-      <div className='pt-[56px] h-screen'>
```
이 부분을 삭제하면 아래 수정내용과 같이 모든 컴포넌트에서 해당 CSS 속성값을 각각 적용해야 합니다. 이렇게 바꾸신 이유가 있을까요?!

parkmihyunn replied:
splash에서 h-screen 값때문에 계속 필요없는 스크롤이 생기더라고요 ! 그래서 빠르게 숙숙숙,,, 수정하려다보니 그렇게 됐습니다,,, 하하

hazzun replied:
미현님 리뷰가 늦어져서 죄송합니다 
해당부분 제가 flex-1 값으로 처리해서 각각 컴포넌트에 적용안해도 되게끔 수정했습니다..!

따라서 `className='pt-[56px] h-screen'` 이 부분 수정내용 되돌려서 다시 피알보내주시면 감사하겠습니다!

parkmihyunn replied:
저 부분 수정내용 되돌렸습니다 : )

parkmihyunn committed:
Fix: 변경 녹음 및 재생 함수

1. 녹음 관련 함수를 변경하였습니다
 -> 바보같이,,, 상태를 반대로 설정해놔서 거꾸로 보여지고 있었습니다
2. 재생 관련 함수를 변경하였습니다.
 -> 기존에 다시 누르면 멈추지 않고 다시 재생돼서 겹치는 문제가 있었고, 관련 핸들링 안해놨어서 변경하였습니다.

parkmihyunn committed:
Design: 변경 app.jsx css

1. 복구시켜두었습니다.

hazzun reviewed (approved):
머지 진행시켜
